governing-documents/inactivity.md

```diff
@@ -9,24 +9,24 @@ nav_order: 8
 
-As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for moving maintainers to emeritus status when a project does not have an explicit policy of their own, and the responsibilities of the Technical Advisory Council (TAC), Hyperledger Foundation staff, and maintainers in this process.
+As part of the normal lifecycle of a project, codebases and maintainers come and go. This document outlines a LF Decentralized Trust-wide policy for moving maintainers to emeritus status when a project's TSC does not have an explicit policy of its own, as well as the responsibilities of the Technical Advisory Council (TAC), LF Decentralized Trust staff, and maintainers in this process.
 
 ## Maintainer Inactivity
 
 This policy applies to projects that do not have an explicit maintainer inactivity policy. Where a project has an established and functioning policy, only that project's policy will apply.
 
-Hyperledger very much appreciates the contributions of all maintainers but removing write privileges is in the interest of an orderly and secure project.
+LF Decentralized Trust greatly appreciates the contributions of all maintainers; however, removing write privileges is in the interest of maintaining an orderly and secure project.
 
-Activity can be code contributions, code reviews, issue reporting, or any other such activity trackable by GitHub attributed to a Hyperledger repository.
+Activity can include code contributions, code reviews, issue reporting, or any other trackable activity attributed to a LF Decentralized Trust repository on GitHub.
 
-When a maintainer has not had any activity in a particular project for three months they will receive a notification informing them of the inactivity policies. The means and manner of notification (email, github mentions, etc.) will be at the discretion of the TAC Chair or who the TAC Chair designates. 
+When a maintainer has not had any activity in a particular project for three months, they will receive a notification informing them of the inactivity policies. The means and manner of notification (e.g., email, GitHub mentions) will be at the discretion of the TAC Chair or their designee.
 
-When a maintainer has not had any activity in a particular project for six months a proposal will be opened up to move the maintainer from active status to emeritus status. A member of the TAC or a Hyperledger staff member will open this proposal. Any permissions to approve pull requests or commit code and any other such privileges associated with maintainer status will be removed.
+When a maintainer has not had any activity in a particular project for six months, a proposal will be opened to move the maintainer from active status to emeritus status. This proposal will be initiated by a member of the TAC or LF Decentralized Trust staff. All permissions to approve pull requests, commit code, and other related privileges will be removed.
```

Comment:
I recommend that we add "a member of the project's TSC". It is probably more important that the project is responsible for this.